python/cucim/requirements-test.txt

```diff
@@ -1,9 +0,0 @@
-GPUtil>=1.4.0
-imagecodecs>=2021.6.8
-openslide-python>=1.3.0
-opencv-python-headless>=4.6
-psutil>=5.8.0
-pytest>=6.2.4
-pytest-cov>=2.12.1
-pytest-lazy-fixture>=0.6.3
-tifffile>=2022.7.28
```

Comment:
These are already contained in `pyproject.toml`'s optional `test` dependencies

https://github.com/rapidsai/cucim/blob/bf5a6f765de64b522ac060f92e24c52f69834c56/python/cucim/pyproject.toml#L61-L75